src/common/constants/index.ts

```diff
@@ -0,0 +1,49 @@
+export const CATAGORY: string[] = ["한식", "중식", "일식", "양식", "분식", "카페", "편의점", "기타"];
```

Comment:
오타 있네요!
```suggestion
export const CATEGORY: string[] = ["한식", "중식", "일식", "양식", "분식", "카페", "편의점", "기타"];
```

Reply:
타입을 더 좁혀서 표현할 수 있을 것 같아요~
```suggestion
export const CATAGORY = ["한식", "중식", "일식", "양식", "분식", "카페", "편의점", "기타"] as const;
```